worlds/generic/docs/setup_en.md

```diff
@@ -47,39 +50,142 @@ slot name and, if the relevant game requires it, files to associate it with that
+in the `Players` folder. If the folder does not exist then it can be created manually. The files here should not be
+compressed.
 
-After filling the `Players` folder the `ArchipelagoGenerate.exe` program should be run in order to generate a
-multiworld. The output of this process is placed in the `output` folder.
+After filling the `Players` folder, the `ArchipelagoGenerate.exe` program should be run in order to generate a
+multiworld. The output of this process is placed in the `output` folder (usually named something like `AP_XXXXX.zip`).
 
-#### Changing local host settings for generation
+##### Changing local host settings for generation
 
 Sometimes there are various settings that you may want to change before rolling a seed such as enabling race mode,
 auto-release, plando support, or setting a password.
 
-All of these settings plus other options are able to be changed by modifying the `host.yaml` file in the Archipelago
+All of these settings, plus other options, may be changed by modifying the `host.yaml` file in the Archipelago
 installation folder. The settings chosen here are baked into the `.archipelago` file that gets output with the other
-files after generation so if rolling locally ensure this file is edited to your liking *before* rolling the seed.
+files after generation so if rolling locally ensure this file is edited to your liking **before** rolling the seed. This
+file is overridden when running the Archipelago Installation software. If you have changed settings in this file, and
+would like to retain them, you may rename the file to `options.yaml`.
 
 ## Hosting an Archipelago Server
 
-The output of rolling a YAML will be a `.archipelago` file which can be subsequently uploaded to the Archipelago host
-game page. Archipelago host game page: [Archipelago Seed Upload Page](/uploads)
+When a multiworld seed is generated, the multidata will be output as a `.archipelago`. If the game was generated locally,
+A compressed folder will be in `/output` and will contain the `.archipelago`, the spoiler, and any relevant files for the
+generated games.
+
+### Hosting on the website
+
+After a seed page has been created on the website, clicking on `Create Room` will create a new server instance, and a
+page that can be linked to the other players, so they can all see the connection info, obtain their data files, and
+connect to the multiworld. Simply click on the url in the title bar, copy the link, and send it to your friends. The
+person who created the room becomes the "owner" of the room, and as such has access to the server console. Clearing
+cookies will remove access to this console, and there is no way to regain it. If a server password was set when
+generating the multiworld game, server admin priveleges may be gained by entering `!admin <password>` from the
+`ArchipelagoTextClient.exe`.
+
+#### The room page
+
+![Screenshot of Room Page](/static/generated/docs/Archipelago/example_room.png)
+1. Host Name
```

Comment:
Most of the clients ask for "Server" and not "Host Name", this could be changed to match, along with changing the screenshot itself